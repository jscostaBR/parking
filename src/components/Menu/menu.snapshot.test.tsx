import * as React from 'react'
import { Menu } from '.'
import renderer from 'react-test-renderer'

describe('render Menu', () => {
  it('render correctly', () => {
    const tree = renderer.create(<Menu />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
