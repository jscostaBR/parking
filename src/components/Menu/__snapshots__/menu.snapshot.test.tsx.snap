// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Menu render correctly 1`] = `
[
  <div
    className="sc-bczRLJ ccLdSK"
  >
    <img
      alt=""
    />
    <img
      alt=""
      className="hamburger"
      onClick={[Function]}
    />
  </div>,
  <div
    className="sc-gsnTZi feLnsf"
  >
    <div
      className="header"
    >
      <img
        className="logo"
      />
      <img
        className="close"
        onClick={[Function]}
      />
    </div>
    <div>
      <span>
        Entrada
      </span>
      <span>
        Sa√≠da
      </span>
    </div>
  </div>,
]
`;
